_posts/en/newsletters/2020-08-12-newsletter.md

```diff
@@ -0,0 +1,206 @@
+  [resumed][myers anyprevout] a discussion about
+  [SIGHASH_ANYPREVOUT][topic sighash_noinput] and provided a nice
+  diagram of how its two signature hash (sighash) types would be used
+  within [eltoo][topic eltoo].  He also asked several questions about
+  minimizing the number of network round trips needed to create eltoo
+  state updates.  These questions received [answers from
+  ZmnSCPxj][zmnscpxj reply] but also sparked a [second
+  discussion][corallo relay] about attacks against LN payment atomicity
+  within the context of eltoo.
+
+    The advantage of eltoo over the currently used LN-penalty mechanism
+    is that the publication of old eltoo channel states onchain doesn't
+    prevent the publication of the ultimate channel state.  This is
+    achieved in eltoo by creating signatures using `SIGHASH_ANYPREVOUT`
+    so that the signature from the ultimate state can spend bitcoins
+    from either the initial state, the penultimate state, or any state
+    in between.  However, transactions still need to identify which
+    state (previous output) they are attempting to spend.
+
+    One problem with that mechanism is that both an attacker and an
+    honest user could both try spending from the same previous state.
+    Miners and relay nodes would only keep one of those transactions in
+    their mempool, and there may be ways the attacker could ensure it
+    was an old state.  Another problem is that the attacker could
+    possibly trick the honest user into spending from a state that relay
+    nodes hadn't seen and so relay nodes would possibly reject the
+    unconfirmed transaction.  Although neither of these problems would
+    prevent the ultimate state from eventually being confirmed onchain,
+    they could be used for [transaction pinning][topic transaction
+    pinning] to prevent one of the honest user's time-sensitive
+    transactions from confirming in time.  These problems are similar to
+    the attack against LN payment atomicity described in [Newsletter
+    #95][news95 ln atomicity].  One proposed mitigation would be to have
+    special nodes (perhaps part of LN routing software) that looked for
+    cases where eltoo was being used and could use their knowledge of
+    that protocol to tell miners which transaction would truly be most
+    profitable to mine.
+
+## Field report: How segwit and batching could have saved half a billion dollars in fees
+
+{% include articles/veriphi-segwit-batching.md %}
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+[Implement ADDRv2 support (part of BIP155)][review club #19031] is a PR
+([#19031][Bitcoin Core #19031]) by Vasil Dimov that proposes a partial
+implementation of the [BIP155][] `addrv2` P2P message.
```

Comment:
This gives the impression that the PR partially implements `addrv2` whereas it fully implements `addrv2` (and partially BIP155, for full BIP155 we need TORv3, I2P, CJDNS implemented too, using the `addrv2` messages).

So, maybe reword to: ... proposes a partial implementation of BIP155 - the `addrv2` P2P message which would enable nodes to send and receive TORv3, IP2 and other larger addresses.

Reply:
Maybe the shorter "partial implementation of BIP155." We go into the details further down.